Update pricing.md (#49)
Kodular/Documentation: docs/pricing.md

Kodular Creator is **completely free of charge**. You can use it for whatever you want, and _we will never ask you to pay us_.

> We wanted it to be like that so **anyone could convert their awesome ideas into apps** in a simple way. We believe that _everybody should have the right to create apps even without knowing how to code_. That's why, our platform is free and **it will remain like that forever**.

Reviewer: ```suggestion
We wanted it to be like that so **anyone could convert their awesome ideas into apps** in a simple way. We believe that _everybody should have the right to create apps even without knowing how to code_. That's why our platform is free and **it will remain like that forever**.
```

No need for a comma there

This also extends to **Kodular Store** and **Kodular Extensions IDE**.

!!! quote "Why this business model?"
    Kodular was created to provide an accessible suite for everyone to learn and, later, professionally create apps. Due to this, we strongly believe learning to code has to be a right and nobody should have to pay in order to learn the basics of programming.  

>     Kodular Creator provides one of the most complete tools in Internet to create apps without coding, and moreover it is being used by tons of developers from all over the world. Everybody should be able to make their ideas a reality, independently of their income. Someone can have the next big revolutionary idea, but sometimes without money it cannot be done.  

Reviewer: ```suggestion
    Kodular Creator provides one of the most complete tools in Internet to create apps without coding, and moreover it is being used by tons of students from all over the world. Everybody should be able to make their ideas a reality, independently of their income. Someone can have the next big revolutionary idea, but sometimes without money it cannot be done.  
```

Keep students

Author: IMO, developers fits there better. Kodular is for everyone wanting to develop apps, and hence, isn't limited to the students.

Author: @barreeeiroo

Author: I've already committed other suggestions proposed by you. Any thoughts on this one?

    We don’t want this to happen.  
    **What is fairer? Getting a fixed quota from everyone, without taking into account if they earn something, or taking the proportional earnings people make with our platform?** We think the answer is very obvious.  

>     We **DO NOT** earn a single penny from the apps that don’t earn anything with our platform. We only earn from the apps that are monetized with ads or IAP.

Reviewer: ```suggestion
    We **DO NOT** earn a single penny from the apps that don’t earn anything with our platform. We only earn from the apps that are monetized with ads using Kodular Creator.
```

We don't get money either from IAP

## Commissions

!!! tip "Updated: 19 January 2019 - 02 May 2019"
    Due to some technical issues with Google, apps generated with Kodular Creator Versions from Draco 1.3.1 to 1.3B.4, both included, will be free of comission. We will update this page as soon as it gets fixed.

To keep our services free for everyone, forever, _we take a small percentage of commission from your earnings in ads_. We do not force anybody to place ads in apps, but if you want to earn money, we take a percentage of your earnings.

> The ads from which we take commissions are **only** Interstitial ones. All other ad types are free of commissions

Reviewer: ```suggestion
The ads from which we take commissions are **only** Interstitial and Rewarded Video ones. All other ad types are free of commissions
```
